.github/actions/backup-and-restore-snapshot-database/action.yml

```diff
@@ -39,7 +39,7 @@ runs:
     - name: Backup database
       shell: bash
       run: |
-        bin/konduit.sh cpd-ecf-${{ inputs.environment }}-web -- pg_dump -E utf8 --compress=1 --clean --if-exists --no-privileges --no-owner --verbose -f backup-${{ inputs.environment }}.sql.gz
+        bin/konduit.sh cpd-ecf-${{ inputs.environment }}-web -- pg_dump -E utf8 --exclude-table-data api_requests --exclude-table-data versions --exclude-table-data email_schedules --exclude-table-data emails --compress=1 --clean --if-exists --no-privileges --no-owner --verbose -f backup-${{ inputs.environment }}.sql.gz
```

Comment:
I think in the snapshot we need to include those as they help us debug especially the api request table so we shouldn't exclude them I don't think

Reply:
@cwrw I know that and totally agree and It won't be merged.. as just I added to be able to refresh snapshot quickly and test the script.